client/src/pages/Home.jsx

```diff
@@ -0,0 +1,39 @@
+import Footer from "../components/Footer";
+import HomeCards from "../components/HomeCards";
+import Header from "../components/Header";
+import Slogan from "../components/Slogan";
+import "../styles/app.scss";
+
+function Home() {
+  return (
+    <>
+      <header>
+        <Header planetName="Neptune" />
```

Comment:
The component name isn't ideal, or you'd have to use the header tag in the component.